### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by SoftRuby** [AlizerUncaged/desktop-waifu#14]
when I run the start program it sowing this "No Python at 'C:\Users\flandre-real\AppData\Local\Programs\Python\Python310\python.exe' "
![Untitled](https://github.com/AlizerUncaged/desktop-waifu/assets/103818745/ac7ef7a8-6901-481d-befe-03d57fbf5aea)

**Comment by Yoishima** [comment]
go to desktop-waifu-master/waifu and look for pyvenv.cfg and change the directory to where your python.exe is installed

**Comment by SoftRuby** [comment]
> go to desktop-waifu-master/waifu and look for pyvenv.cfg and change the directory to where your python.exe is installed

Thx for help

**Comment by SoftRuby** [comment]
> go to desktop-waifu-master/waifu and look for pyvenv.cfg and change the directory to where your python.exe is installed

but bro there is another error is occurring 
![npm install  _character_ai_ 18-05-2023 17_40_52](https://github.com/AlizerUncaged/desktop-waifu/assets/103818745/5f20c330-cea7-4119-a500-c31b3d505dac)

**Comment by Yoishima** [comment]
yeah that's my current problem as well, api key and voicevox are good and valid, but still shows that error

**Comment by Sylensar** [comment]
Did yall get it working? @SoftRuby @Yoishima

**Comment by Yoishima** [comment]
> 

nah, now it's just saying "<empty message>" after pressing right shift+right ctrl, i give up lmao
